### PR TITLE
Review one of my past projects

### PR DESCRIPTION
**Pull request by SSEKPIUS**
Playing the role of the Code Reviewer:
- Check if i have used [semantic HTML tags]
- Check if i applied [HTML & CSS best practices]

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,60 +2,141 @@
...
+    <header id="header">
+      <div id="main-logo">
+        <img id="header-img" src="images/apex-logo.svg" alt="logo"/>
+        <p>Apex Computer Systems</p>
+      </div>
+      <nav id="nav-bar">
+        <div>
+          <a href="#features" class="nav-link">Features</a>
+          <a href="#pricing" class="nav-link">Pricing</a>
+          <a href="#where-to-buy" class="nav-link">Where to Buy</a>
```
Comment (SSEKPIUS) [inline index.html:22]: kindly use ul tags for this section, it makes your code more semantic

File: index.html [hunk 2]
```diff
@@ -2,60 +2,141 @@
...
+
     <link rel="stylesheet" href="css/style.css">
+ 
   </head>
 
   <body>
+    <header id="header">
+      <div id="main-logo">
+        <img id="header-img" src="images/apex-logo.svg" alt="logo"/>
+        <p>Apex Computer Systems</p>
```
Comment (SSEKPIUS) [inline index.html:16]: It's best to use h1 tags for page headers. It makes your page well optimized for search engines.

File: index.html [hunk 3]
```diff
@@ -2,60 +2,141 @@
...
+        <div>
+          <a href="#features" class="nav-link">Features</a>
+          <a href="#pricing" class="nav-link">Pricing</a>
+          <a href="#where-to-buy" class="nav-link">Where to Buy</a>
+        </div>
+      </nav>
+    </header>
+
+<div id="main-div">
+
```
Comment (SSEKPIUS) [inline index.html:28]: kindly wrap content in  Html semantic tags ie. section tags, it makes your code more semantic

File: css/style.scss [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+  /*
+	9. Create a root stacking context
+  */
+  #root, #__next {
+	isolation: isolate;
+  }
+
+//Styling starts here
+
+#header{
```
Comment (SSEKPIUS) [inline css/style.scss:58]: kindly add space between text and { ,  to avoid linters errors.

File: css/style.scss [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+		 }
+	 }
+
+	// #video-div{
+	// 	place-self:center;
+	// 	width:100%;
+		
+		
+	// }
+
```
Comment (SSEKPIUS) [inline css/style.scss:131]: kindy remove un used code,it makes you code look clean and  professional

File: css/style.css [hunk 6]
```diff
@@ -1,19 +1,293 @@
...
+  */
+html, body {
+  height: 100%;
+}
+
+/*
+	Typographic tweaks!
+	4. Add accessible line-height
+	5. Improve text rendering
+  */
```
Comment (SSEKPIUS) [inline css/style.css:27]: kindly remove unused code, it makes your code cleaner and professional

**Review by SSEKPIUS — COMMENTED** [review]
you have done a great job so far, kindly apply the changes requested, and you will be ready to merge.

**Review by Olamarx — APPROVED** [review]
Hi @SSEKPIUS, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**